resdk/resources/base.py

```diff
@@ -235,6 +232,20 @@ def contributor(self):
 
         return self._contributor
 
+    @property
+    def created(self):
+        """Creation time."""
+        if 'created' not in self._original_values:
+            raise ValueError('Instance does not have `created` attribute.')
```

Comment:
Can you add this only to applicable classes?

Reply:
What do you mean? The only classes that inherit from this base class are those that have `created` and `modified` attributes...